_data-prepper/pipelines/configuration/processors/parse-json.md

```diff
@@ -1,20 +1,20 @@
 ---
 layout: default
-title: json
+title: Parse json
 parent: Processors
 grand_parent: Pipelines
 nav_order: 45
 ---
 
-# json
+# Parse json
```

Comment:
In Data Prepper, you specify the processors using a keyword like `parse_json`.

This header should possibly be `parse_json`.

But, we can use `Parse JSON` instead. The term JSON is an acronym and should be uppercase.